internal/hardware/validator.go

```diff
@@ -147,8 +157,8 @@ func (v *validator) DiskIsEligible(ctx context.Context, disk *models.Disk, infra
 	return notEligibleReasons, nil
 }
 
-func (v *validator) IsValidStorageDeviceType(disk *models.Disk, hostArchitecture string) bool {
-	return funk.ContainsString(v.getValidDeviceStorageTypes(hostArchitecture), string(disk.DriveType))
+func (v *validator) IsValidStorageDeviceType(disk *models.Disk, hostArchitecture string, openshiftVersion string, ociPlatformType bool) bool {
```

Comment:
```suggestion
func (v *validator) IsValidStorageDeviceType(disk *models.Disk, hostArchitecture string, openshiftVersion string, isISCSIEnabled bool) bool {
```
nit: Even though this change is specific to oci, I would prefer to make the intention of this parameter clearer for this function and in `getValidDeviceStorageTypes`. WDYT?

Reply:
I think it's OK for now.  The next big change to this code will probably be when iSCSI support will be generic, and this parameter will be removed.